src/components/VideoPlayerMedia.vue

```diff
@@ -7,15 +7,17 @@
       ref="player"
       :class="{ 'display: none;': currentElementRef === 'player2' }"
     ></audio>
-    <video
-      v-else
-      playsinline
-      id="player"
-      ref="player"
-      :poster="viewer.placeholderImg"
-      :class="{ 'display: none;': currentElementRef === 'player2' }"
-      :style="isSplittedView ? 'border-radius: 0.25rem' : 'border-radius: 0'"
-    ></video>
+    <template v-else>
+      <video
+        playsinline
+        id="player"
+        ref="player"
+        :poster="viewer.placeholderImg"
+        :class="{ 'display: none;': currentElementRef === 'player2' }"
+        :style="isSplittedView ? 'border-radius: 0.25rem' : 'border-radius: 0'"
+      ></video>
+      <audio v-if="viewer.enableDRM" id="drm-audio-player" playsinline></audio>
```

Comment:
I was hoping to reuse the audio element that is used when displayAudioOnly is on for DRM but it was difficult with the way the state with reference to video and currentElementRef is set up. Sometimes state.Control.video is acutally the audio element. I am open to possibly refactoring this to not use Refs and have a state.Controls.player that can be both an audio and/or video element but I would like hear thoughts and am open to other ideas.

Reply:
I think having multiple audios is fine for this. I'm not sure it'd be worth the refactor to make it dual purpose.